pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["flit_core >=3.9,<4"]
+requires = ["flit_core >=3.8,<4"]
```

Comment:
FYI, this should "technically" be okay for running in production, but this will likely cause some inconsistencies when building the package.